source/_posts/2016-07-24-top-fashion-accessories-for-men.markdown

```diff
@@ -0,0 +1,33 @@
+# Top Fashion Accessories for Men
+
+Most of the people will concur with the epigram "women's fashion is much more diverse than men!!". But who says accessories are for women only? Here are top 10 accessories that can add flavour and color to any outfit to fit the preferences of any men out there.
+
+## 1. Watch
+A wrist watch is the most conventional accessory a man can invest in. it adds a punch of simplicity as well as style to your personality. it is one of the most acceptable men's accessory. Mechanical, chronograph, leather strap watches are the latest trends this season.
+
+## 2. Tie
+A tie is pretty much one accessory that adds life to any men's costume. Various mutants include bow-tie, necktie, ascots. A tie radiates class and confidence. Pair up your suit with multicolor, printed , textured, silk ties this summer.
+
+## 3. Pocket Squares
+Regardless of this recently added fashion statement pocket squares have become a perfect addition to men's wardrobe. How to pair them up?? well chose a totally contrast pocket square. no matchy-matchy!!GO for a colorful choice that isn't too in-your-face.
+
+## 4. Wallet
+Black and brown leather wallets for men are perennial. The Hobo Millie Clutch and Elephant wallets are gaining popularity these days. Totally cool isnt it?
+
+## 5. Money clip
+This accessory offers a simple and convenient approach to handle your cash. Its graceful and elegant style makes it a great alternative to carry your money. While other men are still confused as to what is better, a wallet or a money clip, you grab your painted m-clip, chrome or silver plated money clip to earn brownie points in impressing ladies out there!
+
+## 6. Card Holders
+Card holders can be a superb approach to speak to a level of professional skill. They also serve as a great way to gift a business associate. Stainless steel and custom fabric card holders are go-go this season.
+
+## 7. Jewellery for men
+Yes unsual it seems,, but jewellery for men is gaining popularity now-a-days. Common jewellery for men is the wedding ring. THe most effective way for a man to carry jewellery is to be comfortable about it. Ear studs, copper bracelets, lapel pins, beaded necklace are some well executed jewellery intended for men.
+
+## 8. Belt
+Belt is a safe and easy pick accessory for men. It no longer serves aa s tool for holding mne's trouser but a great style statement. Wide buckles and a webbed, fabric version with a double-D buckle fastening works well for this summer.
+
+## 9. Sunglasses
+Sunglasses, aviators, shades are one of the accessory men wear on their face. They are a must have for glamorous, High Fashion look. Tortoiseshell frames and sleek metal are latest fashion trends..
+
+## 10. Cuff links
```

Comment:
The article seems to be great and well-researched.
But the author must know that simply decorating the article with heavy words does not mean your article is good, you must add relevant words and such words which are easily understandable by the common audience. Nobody wants to read an article which makes them think twice about words used in it.
You have a good vocabulary but you need to understand the relevancy and make it more human interesting. Although, it is a very good attempt.